host_reference/README.md

```diff
@@ -0,0 +1,28 @@
+# QUDA 1.0.0
+
+## host_reference
+
+This directory contains host side reference code that cross checks the GPU kerels in QUDA
+with host side calculated routines. Currenly supported reference routines for matrix 
+operators are:
+
+	 1. Wilson dslash
+	 2. Clover dslash
+	 3. Staggered dslash
+	 4. Domain Wall dslash
+	 5. Covariant derivative
```

Comment:
Missing twisted mass, non-degenerate twisted mass, twisted clover

Reply:
and also should state 4-d and 5-d Shamir DWF as well as Möbius

Reply:
bfc720c91